db/mongodb/src/main/java/io/helidon/db/mongodb/StatementParsers.java

```diff
@@ -0,0 +1,424 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.db.mongodb;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.Map;
+
+import javax.json.Json;
+
+/**
+ * Statement parameter parsers.
+ */
+final class StatementParsers {
```

Comment:
This needs some code cleanup and also common ancestor with JDBC parser in common.